modules/weko-schema-ui/weko_schema_ui/schema.py

```diff
@@ -714,7 +730,9 @@ def clean_none_value(dct):
                 mpdic = value_item_parent.get(
                     self._schema_name) if self._schema_name \
                     in value_item_parent else ''
-                if isinstance(mpdic, str) and len(mpdic) == 0:
+                if (isinstance(mpdic, str) and len(mpdic) == 0) or (len(
+                    self._ignore_list) > 0 and key_item_parent
+                                                        in self._ignore_list):
```

Comment:
You can reduce this condition
isinstance(mpdic, str) and len(mpdic) == 0
to
mpdic is ""
and
(len(self._ignore_list) > 0 and key_item_parent in self._ignore_list)
to 
self._ignore_list and key_item_parent in self._ignore_list: